### PR TITLE
update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by djskinner**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,10 +17,11 @@ To start a release:
 
 Once the release PR has been approved, merge the PR into `main`. You are now ready to make the release. Ensure you are logged in to npm and that you have access to publish the package.
 
-- Bump the package version:
+- Bump the package version and push the new tag:
 
   ```
   npm version <major|minor|patch>
+  git push origin main
```
Comment (bengourley) [inline CONTRIBUTING.md:24]: does this push the tag? Or do you need to do `git push --tags`?

Comment (djskinner) [reply CONTRIBUTING.md:24]: Yes it pushes the tag and the commit. This is the output from when I did it earlier:

```
~/bugsnag-source-maps main#v2.2.0 ⇡1 ❯ git push origin main                                                                                                                                    13s
Enumerating objects: 8, done.
Counting objects: 100% (8/8), done.
Delta compression using up to 16 threads
Compressing objects: 100% (5/5), done.
Writing objects: 100% (5/5), 929 bytes | 929.00 KiB/s, done.
Total 5 (delta 3), reused 0 (delta 0)
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
To github.com:bugsnag/bugsnag-source-maps.git
   5e2ae99..e7f7634  main -> main
 * [new tag]         v2.2.0 -> v2.2.0
```

Comment (bengourley) [reply CONTRIBUTING.md:24]: I think that might be a local setting of yours? I can't find a watertight answer on it, but this SO thread has some details: https://stackoverflow.com/questions/2988088/do-git-tags-get-pushed-as-well/2988182

Do you have `push.followTags = true` set in your git config or something?

Comment (djskinner) [reply CONTRIBUTING.md:24]: Ah, yes I do, though I didn't change that myself! I guess we would need both `git push --tags` and `git push origin main` then? As we also want the commit to be pushed to origin?

Comment (bengourley) [reply CONTRIBUTING.md:24]: yeah you'd need both `git push origin main` and `git push --tags`

**Review by imjoehaines — APPROVED** [review]

**Pull request merged**